src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import duke.task.Task;
+
+/**
+ * This class represents Duke's list of tasks.
+ */
+public class TaskList {
+    private static final String STORAGE_PATH = "data/duke.txt";
+    private List<Task> list = new ArrayList<>();
```

Comment:
Maybe rename this List<Task> as "tasks"? To represent a list of Task objects.

Reply:
Thanks for spotting this, will rename this.